cds-snc/cppd-medical-report-sails
# Configure automated tests to run in Github Actions

# What this does
Runs our test suite in Github Actions! Once merged, this will run our Cypress test suite against all PRs.

To get this going, I've implemented a testing config that uses a local sqlite database and in-memory session store. This means we can run our application tests quickly and easily in a basic Github Action. 

We can discuss future iterations where we test with the full infrastructure.

## To test
Um, well, just check out the successful test run linked below I guess:

- Click the details link displayed here:
  ![image](https://user-images.githubusercontent.com/1187115/75393082-549e2380-58bb-11ea-9561-45e2fb70878e.png)
- Notice all the green checkmarks, then click on the Cypress tests:
  ![image](https://user-images.githubusercontent.com/1187115/75393143-6d0e3e00-58bb-11ea-983c-2b33d3f8c87c.png)
- Huzzah:
  ![image](https://user-images.githubusercontent.com/1187115/75393182-7e574a80-58bb-11ea-8a38-62316c043cf3.png)
